app/models/host/managed.rb

```diff
@@ -89,7 +89,7 @@ class Jail < ::Safemode::Jail
       :provision_interface, :interfaces, :bond_interfaces, :bridge_interfaces, :interfaces_with_identifier,
       :managed_interfaces, :facts, :facts_hash, :root_pass, :sp_name, :sp_ip, :sp_mac, :sp_subnet, :use_image,
       :multiboot, :jumpstart_path, :install_path, :miniroot, :medium, :bmc_nic, :templates_used, :owner, :owner_type,
-      :ssh_authorized_keys, :pxe_loader, :global_status, :get_status, :puppetca_token
+      :ssh_authorized_keys, :pxe_loader, :global_status, :get_status, :puppetca_token, :last_report
```

Comment:
Does the `ConfigReport` model have a Jail definition?

Reply:
no, this is the method on host object that we're making available, meaning you can do `@host.last_report`

but now when I think about that, macro like `host_last_time(@host)` would probably be better to avoid relying on active record, is that what you thought?

Reply:
I just want to make sure the last_report_object (will be of type ConfigReport::Jail) is actually usable in safemode. I was wondering if we also need to whitelist any methods in `ConfigReport`.

Reply:
last_report does not return config report object, but a timestamp, which is a host attribute.

Reply:
@ares: You're right. Sorry, I misread the code. 👍 from me.

Reply:
clicking the green button appreciated, this can be easily reverted if someone would object :-)